src/docs/arc42/chapters/chap-09-Decisions.adoc

```diff
@@ -43,12 +43,6 @@ Find details on how HSC implements HTML parsing in the {xrefConceptHtmlEncapsula
 
 === String Similarity Checking with https://wikipedia.org/wiki/Jaro%E2%80%93Winkler_distance[Jaro-Winkler-Distance]
 
-The small https://github.com/rrice/java-string-similarity[java-string-similarity] library (by Ralph Allen Rice) contains implementations of several similarity-calculation algorithms. As it is *not available* as public binary,
-we use the sources instead, primarily:
+The small https://github.com/rrice/java-string-similarity[java-string-similarity] library (by Ralph Allen Rice) contains implementations of several similarity-calculation algorithms. As it is public binary, available at central https://central.sonatype.com/artifact/net.ricecode/string-similarity[maven repository] we have used it as external library dependencies.
```

Comment:
I try to follow [one sentence per line](https://nick.groenen.me/notes/one-sentence-per-line/) convention which is not explicitly stated somewhere. But could you please make a line break between "." and next sentence ("As ...")?

Reply:
done